overview.md

```diff
@@ -44,23 +50,26 @@ Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks
+The objective of CVCs is to ensure that they represent MKR voters that are aligned with the values of that CVC. Members of CVCs will be compensated for their engagement under certain conditions, thereby incentivizing active governance. 
+
+CVCs work together with [Delegates](maker-core/delegates.md) as well as [Advisory Councils](maker-core/advisory-councils.md) to implement specific governance changes.
 
 ### [Tokenomics](tokenomics/launch-tokenomics.md)
 
-The Endgame Plan introduces [Elixir](tokenomics/elixir.md) and [SubElixir](tokenomics/subelixir.md), which are liquidity pool tokens representing liquidity pools holding various combinations of MKR, ETH, DAI and SubDAO governance tokens. The aim of these tokens is twofold. First, it ensures alignment of interests between Maker Core and SubDAOs. Second, it also improves liquidity for all these tokens on the open market. 
+The Endgame Plan introduces [Elixir](tokenomics/elixir.md) and [SubElixir](tokenomics/subelixir.md), which are liquidity pool tokens representing liquidity pools holding various combinations of MKR, Dai and SubDAO governance tokens. The aim of these tokens is twofold. First, it ensures alignment of interests between Maker Core and SubDAOs. Second, it also improves liquidity for all these tokens on the open market. 
 
 The Endgame Plan has two major tokenomics updates. First, annual MKR minting will be used to incubate SubDAOs, fund DVCs, and incentivize the workforce. These emissions counteract the MKR burn and reduce the likelihood of MKR becoming concentrated in the hands of large holders.
 
-ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDAI](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDAI is one of the central products of the Endgame Plan. 
+ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDai](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDai is one of the central products of the Endgame Plan. 
+
+In the final stages of Endgame, Dai will turn into an Unbiased World Currency. It may not remain pegged to the US dollar and may instead either peg to another reference currency, or become free-floating. In the latter case, incentive mechanisms will determine Dai's target price. 
 
-In the final stages of Endgame, DAI will turn into an Unbiased World Currency. In particular, it is not expected to remain pegged to the US Dollar and will instead have alternative mechanisms that determine its Target Price. 
+Maker Governance no longer guarantees that MKR tokens will function as a backstop in the event that Maker is insolvent. Governance may choose to change the Target Price of Dai to reflect this. 
```

Comment:
Feel like you should reword this, as it's summarizing some pretty high-controversy stuff if it ever happens. Maybe:

"Maker Governance no longer guarantees that the MKR token will function as a backstop in the event that Maker becomes insolvent. Governance may instead choose to adjust the Target Price of Dai which would result in a loss in value for all Dai holders."